.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: java
+
+jdk:
+  - oraclejdk7
+  - oraclejdk8
+
+install: mvn -f gson install -DskipTests=true
+script: mvn -f gson test
```

Comment:
Do you mean move the sub-projects out as their own projects? What would you suggest?